utils/auth.py

```
@@ -0,0 +1,76 @@
+    """
+    JWT 토큰을 검증하고 페이로드를 반환합니다.
+
+    Args:
+        token: JWT 토큰 문자열
+
+    Returns:
+        토큰에 담긴 데이터 또는 None
+    """
+    try:
+        payload = jwt.decode(token, SECRET_KEY, algorithms=[ALGORITHM])
+        return payload
```

Ruff (0.14.13)

Consider moving this statement to an `else` block

(TRY300)